test/exchange_adapters/xignite.test.ts

```diff
@@ -0,0 +1,117 @@
+import { Exchange, ExternalCurrency } from '../../src/utils'
+
+import BigNumber from 'bignumber.js'
+import { ExchangeAdapterConfig } from '../../src/exchange_adapters/base'
+import { XigniteAdapter } from '../../src/exchange_adapters/xignite'
+import { baseLogger } from '../../src/default_config'
+
+describe('Xignite adapter', () => {
+  let adapter: XigniteAdapter
+
+  const config: ExchangeAdapterConfig = {
+    baseCurrency: ExternalCurrency.EUR,
+    baseLogger,
+    quoteCurrency: ExternalCurrency.XOF,
+  }
+
+  beforeEach(() => {
+    adapter = new XigniteAdapter(config)
+  })
+
+  afterEach(() => {
```

Comment:
This block should be removed. We're not using any mock functions or timers so these calls are unnecessary. If anything this may be slightly misleading for people looking at this in future.

Reply:
yeah good catch, I had some tests that were using this at some point and then forgot to remove them